1-exercises/A-this/exercise-1.js

```diff
@@ -15,7 +15,9 @@ let person = {
     name: "Adrian",
     age: 32,
     currentAddress: "London",
-    celebrateBirthday: /* todo */
+    celebrateBirthday: function(){
+         this.age;
```

Comment:
Good job Zaw. The exercise ask us to incerment age of person by 1
in your code by this.age it will return 32 however we expected 33.
you can use age++ or increment it by 1.